libcontainer/seccomp/seccomp_linux.go

```diff
@@ -151,17 +151,20 @@ func matchCall(filter *libseccomp.ScmpFilter, call *configs.Syscall) error {
 		return errors.New("empty string is not a valid syscall")
 	}
 
-	// If we can't resolve the syscall, assume it's not supported on this kernel
-	// Ignore it, don't error out
-	callNum, err := libseccomp.GetSyscallFromName(call.Name)
+	// Convert the call's action to the libseccomp equivalent
+	callAct, err := getAction(call.Action, call.ErrnoRet)
 	if err != nil {
+		return fmt.Errorf("action in seccomp profile is invalid: %w", err)
+	}
+	if callAct == defAct {
 		return nil
 	}
 
-	// Convert the call's action to the libseccomp equivalent
-	callAct, err := getAction(call.Action, call.ErrnoRet)
+	// If we can't resolve the syscall, assume it's not supported on this kernel
+	// Ignore it, don't error out
+	callNum, err := libseccomp.GetSyscallFromName(call.Name)
 	if err != nil {
-		return fmt.Errorf("action in seccomp profile is invalid: %w", err)
+		return nil
```

Comment:
How about add some warning log?

Reply:
What you request is a new functionality, not really related to the fix, but yes, it makes sense.

Added as a separate commit, PTAL.

Reply:
Thanks

Reply:
This shouldn't be a warning, because then you'll end up with a bunch of warnings on old kernels that serve no purpose. At best this should be a debug message.

Reply:
Done.